assets/app/views/browse/edit-build-config.html

```diff
@@ -0,0 +1,257 @@
+                        entries="envVars"
+                        delimiter="="
+                        key-validator="env"
+                        delete-policy="always"
+                        key-title="Name"
+                        readonly-keys=""
+                        key-validation-tooltip="A valid environment variable name is an alphanumeric (a-z and 0-9) string beginning with a letter that may contain underscores."></osc-key-values>
+
+                    </div>
+                  </div>
+                <div class="col-lg-6">
+                  <div class="section">
+                    <h3>Triggers</h3>
+                    <dl class="dl-horizontal left">
+                      <div class="checkbox">
+                          <label>
+                            <input type="checkbox" ng-model="pickedTriggers['webhook']"/>
+                            Enable the webhook build trigger
+                            <span class="help action-inline">
+                              <a href data-toggle="tooltip" data-placement="right"
+                                 data-original-title="The source repository must be configured to use the webhook to trigger a build when source is committed.">
+                                <i class="pficon pficon-help"></i>
+                              </a>
+                            </span>
+                          </label>
+                        </div>
+                        <div class="checkbox">
+                          <label>
+                            <input type="checkbox" ng-model="pickedTriggers['imageChange']"/>
+                            Automatically build a new image when the builder image changes
+                            <span class="help action-inline">
+                              <a href data-toggle="tooltip" data-placement="right" data-original-title="Automatically building a new image when the builder image changes allows your code to always run on the latest updates.">
+                              <i class="pficon pficon-help"></i>
+                              </a>
+                            </span>
+                          </label>
+                        </div>
+                        <div class="checkbox">
+                          <label>
+                            <input type="checkbox" ng-model="pickedTriggers['configChange']"/>
+                            Automatically build a new image when the build configuration changes
+                          </label>
+                        </div>
+                    </dl>
+                  </div>
+                </div>
+              </div>
+                <div class="buttons gutter-top-bottom gutter-top-bottom-2x">
+                  <!-- unable to use form.valid.  need to fix validators in labels and key values directive -->
+                  <a class="btn btn-primary btn-lg"
+                      ng-click="save()"
+                      ng-disabled="form.invalid"
```

Comment:
Does `ng-disabled` work on a link? It might be better to make this a button anyway so that enter submits the form.